eldritch/encounters.py

```diff
@@ -136,10 +136,27 @@ def Administration7(char):
   gain = events.Gain(char, {"dollars": 8})
   arrest = events.Arrested(char)
   return events.PassFail(char, check, gain, arrest)
+def Asylum1(char):
+  check = events.Check(char, "lore", 0)
+  loss = events.Loss(char, {"sanity": 1})
+  gain1 = events.Gain(char, {"clues": 1})
+  roll0 = events.Sequence([loss, gain1], char)
+  roll1 = events.Gain(char, {"clues": 2})
+  roll3 = events.Gain(char, {"clues": 3})
+  cond = events.Conditional(char, check, "successes", {0: roll0, 1: roll1, 3: roll3})
+  return events.Sequence([check, cond], char)
 
```

Comment:
I know I have some style nitpicks later about two blank lines between top-level definitions, but I don't know that it makes a lot of sense here - pick any standard you want for the encounters file (okay, not any standard - 0, 1, or 2 lines between encounter definitions), and let's make the whole file consistent with the standard you pick.